WTWSMS/common/buildings/WtWSMS_historical_castle.txt

```diff
@@ -155,34 +155,13 @@ castle = {
 		build_time = 0
 
 		culture_techpoints = 0.05
+		galleys = 50
 		
 		ai_creation_factor = 100
 		
 		extra_tech_building_start = 0.0
 	}
 
-	ca_carthport_1 = { # Expanded Port of Carthage, credits to Morloran
-		potential = {
-			FROMFROM = { title = b_cartaghe }
-		}
-		desc = ca_carthport_desc
-		port = yes
-		trigger = { TECH_NAVAL = 2 }
-		prerequisites = { ca_carthport_0 }
-		upgrades_from = ca_carthport_0
-		build_cost = 100
-		build_time = 1460
-
-		culture_techpoints = 0.05
-		galleys = 50
-		liege_prestige = 0.5
-		monthly_character_prestige = 0.5
-
-		ai_creation_factor = 0
-		
-		extra_tech_building_start = 8.0
-	}
-
```

Comment:
Why did you delete `carthport_1`? I think it makes sdense to have it, given that the port experienced a decline.